File: Python/PGS/LEVEL1/수박수박수박수박수박수/gunwoo.java
package Python.PGS.LEVEL1.수박수박수박수박수박수;

class Solution {
    public String solution(int n) {
        StringBuilder answer = new StringBuilder();
        for(int i = 0; i < n; i++){
            if(i % 2 != 0){
                answer.append('박');
            }
            else{
                answer.append('수');
            }
        }
        return answer.toString();
    }
}

File: Python/PGS/LEVEL2/피보나치 수/gunwoo.py
def solution(n):
    answer = 0
    if n == 0:
        answer = 0
    elif n == 1:
        answer = 1
    a,b = 0,1
    for i in range(2, n+1):
        a,b = b, a+b
    answer = b%1234567
    return answer